src/modules/aws_ecr/index.ts

```diff
@@ -583,7 +580,7 @@ class RepositoryPolicyMapper extends MapperBase<RepositoryPolicy> {
         ctx,
         this.module.repository.generateId({ repositoryName: rp.repositoryName, region }),
       ));
-    out.policyText = rp?.policyText?.replace(/\n/g, '').replace(/\s+/g, ' ') ?? null;
+    out.policy = JSON.parse(rp?.policyText?.replace(/\n/g, '').replace(/\s+/g, ' ') ?? null);
```

Comment:
Do we still need the `replace` calls here if you're `JSON.parse`ing it?

Also, on `JSON.parse` failure it `throw`s instead of returning something we'd replace with `null`, so you may want to move [`safeParse`](https://github.com/iasql/iasql/blob/main/src/modules/aws_acm/rpcs/common.ts#L1) into the `src/services` directory and use that instead?